train.py

```diff
@@ -72,8 +72,13 @@ def train(hyp, opt, device, tb_writer=None):
         print('Transferred %g/%g items from %s' % (len(state_dict), len(model.state_dict()), weights))  # report
     else:
         model = Model(opt.cfg, ch=3, nc=nc).to(device)  # create
-
+    
     # Freeze
+    if opt.freeze !='':
+        freeze=[]
+        [freeze.append(x)for x in opt.freeze.split()]
```

Comment:
I think it would split using comma. Could be shorten to "freeze=opt.freeze.split(',')"? It actually could be rewritten to be a one liner into Line 82 directly.

Reply:
@NanoCode012 it would split with spacebar:- [Reference](https://www.w3schools.com/python/ref_string_split.asp). This was suggested in  Fixes #679 . Adding argparser would be convenient method to add freezing.

Reply:
Hi, I understand what you mean, I mean this case, `--freeze conv,bn` , wouldn't it make sense using a comma to split? I am not sure if argparser handles spacing? Please correct me if I'm wrong.

Reply:
![Screenshot from 2020-08-11 18-37-37](https://user-images.githubusercontent.com/55156409/89900866-ebf4fa00-dc01-11ea-83a5-e4fd7838e8c8.png)
@NanoCode012 It is working fine, but if you want I will update this as well

Reply:
It's up to you and glenn. I am just giving my opinion with that second point. I just think that shorter will be better in this case.

Reply:
`python train.py --img 640 --batch 16 --epochs 2 --data '../data.yaml' --cfg yolov5s.yaml --weights yolov5s.pt --freeze 'model.0.conv.conv.weight model.0.conv.bn.weight' --nosave --cache`. I just ran this and helped me a lot. @NanoCode012 even I find this an excellent option, this would make our lives easier. Let @glenn-jocher decide weather to merge or not.